persistence-jdbc/core/src/main/resources/reference.conf

```diff
@@ -6,12 +6,32 @@ jdbc-defaults.slick {
   # set to one of: slick.jdbc.PostgresProfile$, slick.jdbc.MySQLProfile$, slick.jdbc.OracleProfile$ or slick.jdbc.H2Profile$
   # profile = "slick.jdbc.PostgresProfile$"
 
-  # The JNDI name
-  jndiName = "DefaultDS"
+  jndiDbName = "DefaultDB"
+
+  async-executor {
+    # number of objects that can be queued by the async exector
+    queueSize = 10000
+
+    # 5 * number of cores
+    numThreads = 20
+
+    # number of threads
+    minConnections = 20
+
+    # 5 * number of threads
+    maxConnections = 100
+  }
+}
+
+
+db.default {
+
+  hikaricp {
+    minimumIdle = ${jdbc-defaults.slick.async-executor.minConnections}
+    maximumPoolSize = ${jdbc-defaults.slick.async-executor.maxConnections}
+  }
 }
 
-# Configure the default database to be bound to JNDI
-db.default.jndiName=DefaultDS
```

Comment:
I think we still want Play to bind the DS to JNDI for JPA to use.

Reply:
I'm binding it back on the JPA reference file. I didn't want to keep things here that are not used.